test/e2e/transformer/test_transformer_odh.py

```
@@ -0,0 +1,535 @@
+    finally:
+        try:
+            cleanup_sa(kserve_client, sa_name, KSERVE_TEST_NAMESPACE)
+            if not _should_skip_deletion(test_failed):
+                kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+            elif test_failed:
+                logger.info(
+                    "Skipping deletion of %s due to test failure "
+                    "(SKIP_DELETION_ON_FAILURE=True)",
+                    service_name,
+                )
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)